apps/bfd-pipeline/bfd-pipeline-rif-load/src/main/java/gov/cms/bfd/pipeline/rif/load/RifLoader.java

```diff
@@ -79,6 +85,8 @@
    */
   private static final int RECORD_BATCH_SIZE = 100;
 
+  private static final int MAX_FILE_AGE = 30; // Days
```

Comment:
In some past projects I would put these in configs. This project doesn't have many configs which can be a virtue because it is simpler to operate. This is especially true since we are practicing "infrastructure as code".